README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
+            "section": "Databases",
+            "depends": [
+              "python"
+            ],
+            "suggests": [
+              "libgnome-keyring0"
+            ]
+          },
+          "redhat_config": {
+            "category": "Applications/Databases",
+            "requires": [
+              "libXScrnSaver(x86-64)"
```

Comment:
@imlucas Though if you'd like me to, please create a JIRA ticket to look into exactly what depends on `libXScrnSaver(x86-64)` as we've got a pretty full sprint this week. 

Hopefully it will be visible from the `ldd mongodb-compass-dev` output on Red Hat 7, otherwise happy to have an excuse to go hunting to see what's it's linked to in our dependency tree.